frame/utility/src/lib.rs

```diff
@@ -100,6 +105,9 @@ pub mod pallet {
 
 		/// Weight information for extrinsics in this pallet.
 		type WeightInfo: WeightInfo;
+
+		/// Filtering calls.
```

Comment:
Would be worth mentioning that this is applied to inner dispatches of certain calls, and how it related to `system`'s base-call filter.